salt/elife-libraries/init.sls

```diff
@@ -20,6 +20,7 @@ pattern-library-gulp:
 make:
     pkg.installed
 
+# TODO: what do we have that requires ruby?
```

Comment:
`pattern-library` use to depend on [`gem install`](https://github.com/elifesciences/pattern-library/commit/3f03809c9139f96b7e16d853b15229e39ace9650#diff-5bfe6469d114ea64da22bf52bf88e077R27) (Ruby package manager) but it has since been moved to containers and eventually removed, to my knowledge. Might as well remove this state (conditionally for 18.04) as even if it was installed it would probably be an incompatible version.